aea/helpers/acn/agent_record.py

```diff
@@ -119,18 +139,29 @@ def message(self) -> bytes:
         return self._message
 
     @property
-    def ledger_id(self) -> str:
+    def identifier(self) -> SimpleIdOrStr:
+        """Get the identifier."""
+        return self._identifier
+
+    @property
+    def ledger_id(self) -> SimpleIdOrStr:
         """Get ledger id."""
         return self._ledger_id
```

Comment:
If the above is accepted:

```suggestion
        return str(self._ledger_id)
```

Reply:
but with the second alternative `str(SimpleId(...))` then not needed